templates/userauth/account/all_user_chats.html

```diff
@@ -1,109 +1,129 @@
-                                                                                    from messaging you?
-                                                                                </p>
-                                                                                <p class="text-meta text-black-large pb-6">
-                                                                                    This action cannot be undone.
-                                                                                </p>
-                                                                                <div class="flex gap-4.5">
-                                                                                    <button @click="overlay = ! overlay; console.log('Back button triggered overlay'); $event.preventDefault();"
-                                                                                            class="button button-on-colour">
-                                                                                        Back
-                                                                                    </button>
-                                                                                    <button
-                                                                                            hx-post="{% url 'block_user_chat' user_chat.user.uuid %}"
-                                                                                            hx-trigger="click"
-                                                                                            hx-target="#{{ user_slug }}"
-                                                                                            hx-swap="outerHTML"
-                                                                                            @click="overlay = ! overlay; console.log('RED remove button triggered overlay'); $event.preventDefault();"
-                                                                                            name="action"
-                                                                                            value="remove_swimmer"
-                                                                                            class="button button-danger">
-                                                                                        Block
-                                                                                    </button>
-                                                                                </div>
+            <div class="flex gap-4 grow overflow-hidden">
+                <div class="flex flex-col w-full min-w-[360px] desktop:max-w-[360px] overflow-y-auto {% if page_obj %}hidden{% endif %} desktop:block">
+                    {% include "partials/vertical-spacer.html" with space="4" %}
+                    {% for user_chat in user_chats %}
+                        {% user_chat_slug user_chat as user_slug %}
```

Comment:
Nice new `user_chat_slug` tag to avoid those ugly nested `with` tags.

Reply:
nice